internal/cmd/egctl/envoy_stats.go

```diff
@@ -124,7 +129,7 @@ func newEnvoyStatsCmd() *cobra.Command {
 			return nil
 		},
 	}
-	statsConfigCmd.PersistentFlags().StringVarP(&outputFormat, "output", "o", summaryOutput, "Output format: one of json|yaml|prom")
```

Comment:
previous is intend, should change the description.

Reply:
We support three types json/yaml/prom, the default value should be set with `prometheusOutput`, and remove the unused `summaryOutput`. What should we update here? I do not get it.

Reply:
we don't need support json/yaml on stats.

Reply:
Not sure about that, so why adding json/yaml before cc @tanujd11 ?